paket.dependencies

```diff
@@ -17,7 +17,7 @@ nuget Giraffe
 
 group Build
     source https://api.nuget.org/v3/index.json
-    framework: netstandard2.0
+    framework: net6.0
```

Comment:
Had to change this and remove asyncs from build.fsx, otherwise Fake just refuses to work. No idea what to do about it.

Reply:
I will be removing the `build.fsx` at some point and start using a simple console app for build automation